src/Sign.Cli/Program.cs

```diff
@@ -23,6 +23,13 @@ internal static async Task<int> Main(string[] args)
             Kernel32.LoadLibraryW($@"{baseDirectory}\wintrust.dll");
             Kernel32.LoadLibraryW($@"{baseDirectory}\mssign32.dll");
 
+            if (!Environment.Is64BitProcess)
+            {
+                Console.Error.WriteLine("Only Windows x64 is supported at this time.  See https://github.com/dotnet/sign/issues/474 regarding Windows x86 support.");
+
+                return ExitCode.Failed;
+            }
+
```

Comment:
should we add this logic to the beginning of this method?

Reply:
My reluctance in moving this to the beginning of this method is explained [here](https://github.com/dotnet/sign/pull/476/files#diff-4e7543fbcd4b76878fbcb858fe3748c5df8151894f795bb1db6eafa2381ee391L16-L20).

Presently, I don't have a way of testing x86 Windows to see if this matters.  Azure DevOps only supports bring-your-own-x86-OS; the marketplace doesn't have x86 images available.